src/stores/game.js

```diff
@@ -160,7 +166,10 @@ export const useGameStore = defineStore('game', {
       return state.turn % 2 === state.myPNum;
     },
     hasGlassesEight: (state) => {
-      return state.player.faceCards.filter((card) => card.rank === 8).length > 0;
+      if (!state.player) {
+        return false;
+      }
+      return state.player.faceCards?.filter((card) => card.rank === 8).length > 0 ?? null;
```

Comment:
This should coalesce to false not null so that the type is always a boolean
```suggestion
      return state.player.faceCards?.filter((card) => card.rank === 8).length > 0 ?? false;
```